### PR TITLE
Enhancement: Configure binary_operator_spaces fixer to align with minimal space

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] configures the `binary_operator_space` fixer to align with minimal space
* [x] runs `make cs`

Follows #600.

### CODE REVIEW

File: classes/Http/Form/SignupForm.php [hunk 1]
```diff
@@ -72,17 +72,7 @@ public function validateAll($action = 'create')
...
-            $validFirstName     &&
-            $validLastName      &&
-            $validCompany       &&
-            $validTwitter       &&
-            $validUrl           &&
-            $validSpeakerInfo   &&
-            $validSpeakerBio    &&
-            $validSpeakerPhoto  &&
-            $agreeCoc;
+            $validEmail && $validPasswords && $validFirstName && $validLastName && $validCompany && $validTwitter && $validUrl && $validSpeakerInfo && $validSpeakerBio && $validSpeakerPhoto && $agreeCoc;
```
Comment (localheinz) [inline classes/Http/Form/SignupForm.php]: Interesting twist here.

File: classes/Http/Form/TalkForm.php [hunk 2]
```diff
@@ -67,14 +67,7 @@ public function sanitize()
...
         return
-            $this->validateTitle()       &&
-            $this->validateDescription() &&
-            $this->validateLevel()       &&
-            $this->validateCategory()    &&
-            $this->validateDesired()     &&
-            $this->validateSlides()      &&
-            $this->validateOther()       &&
-            $this->validateSponsor();
+            $this->validateTitle() && $this->validateDescription() && $this->validateLevel() && $this->validateCategory() && $this->validateDesired() && $this->validateSlides() && $this->validateOther() && $this->validateSponsor();
```
Comment (localheinz) [inline classes/Http/Form/TalkForm.php]: And here as well.

**Comment by localheinz** [comment]
@mdwheele 

Conflicts resolved!

**Comment by mdwheele** [comment]
Thanks @localheinz

**Comment by localheinz** [comment]
Thank you, @mdwheele!

**Pull request merged**